guide/interactions/modals.md

```diff
@@ -101,20 +100,42 @@ Restart your bot and invoke the `/ping` command again. You should see a popup fo
 
 Modals are received via an interaction. You can check if a given interaction is a modal by invoking the `#isModalSubmit()` method:
 
-```js {2}
+```js {1,4}
+const { InteractionType } = require('discord.js');
+
 client.on('interactionCreate', interaction => {
-	if (!interaction.isModalSubmit()) return;
+	if (interaction.type !== InteractionType.ModalSubmit) return;
 	console.log(interaction);
 });
 ```
 
+## Responding to modal submissions
+
+The `ModalSubmitInteraction` class provides the same methods as the `CommandInteraction` class. These methods behave equally:
+
+- `reply()`
+- `editReply()`
+- `deferReply()`
+- `fetchReply()`
+- `deleteReply()`
+- `followUp()`
+
+```js{1,4-6}
+client.on('interactionCreate', interaction => {
+	if (!interaction.isModalSubmit()) return;
```

Comment:
```diff
- if (!interaction.isModalSubmit()) return;
+ if (!interaction.type === InteractionType.ModalSubmit) return;
```

Reply:
`if (interaction.type !== InteractionType.ModalSubmit) return;`

Reply:
whoops that's a typo lol, fixing